### DISCUSSION TITLE
사용법 문의

### DISCUSSION THREAD

**Discussion by jj74523** [changdoc/naver-vaccine-macro#43] [Q&A]

**Comment by changdoc** [comment]
네 잔여 백신이 발생하면 예약 신청을 수행합니다.
웨일 우측상단 아이콘은 뭔지 모르겠네요. 아이콘을 눌러보거나 아이콘 위에 마우스를 얹어보면 설명이 나올거 같습니다.

**Reply by jj74523** [reply]
아 네넵 답변감사합니다!
